integration-tests/bats/remotes.bats

```diff
@@ -1463,3 +1463,24 @@ setup_ref_test() {
     dolt push --set-upstream origin feature
     dolt push
 }
+
+@test "remotes: clone local repo and push" {
```

Comment:
Name of this test should be "clone local repo with file url"

Couple things: 

1) Make sure no uncommitted changes show up in the cloned repo.
2) Make sure push works as expected. Push on a different branch than the one that has working set changes.

Reply:
Also make sure this works with an absolute path